recipes/hope/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "hope" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/horta/hope/archive/{{ version }}.tar.gz
+  sha256: 6b0cc309458fd613ddeaee267fdde5693094298e30f15690057f29c3487a5b60
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja  # [win]
+    - python
+  host:
+    - conda-build
```

Comment:
Do you really need conda-build here?!

Reply:
No need indeed, thanks! Just changed it.